src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1543,9 +1543,11 @@ private void visitTypeValueAlias(String unqualifiedName, ObjectType otype) {
       visitTemplateTypes(otype, Collections.emptyList(), false);
       emit(";");
       emitBreak();
-      if (!otype.isInterface()) {
+      if (!otype.isInterface() || !getTypePropertyNamesToEmit(otype, true).isEmpty()) {
```

Comment:
What happens if the original declaration is in a different js_library, i.e. all clutz is seeing is:

```
goog.module('alias_for_interface');

var AliasedInterface = goog.require('aliased_interface');
exports = AliasedInterface;
```

I think we already emit the 'var' declaration because isInterface is false on unknown types, but double-check and maybe comment?

Reply:
Good point, done, and your expectation is correct.